python/ray/_private/worker.py

```diff
@@ -1924,8 +1939,10 @@ def print_to_stdstream(data):
             batches = [data]
         sink = sys.stdout
 
+    # Ignore the prefix if the logging config is set.
+    ignore_prefix = global_worker.get_job_logging_config is not None
```

Comment:
We only need to determine `ignore_prefix` once instead of every time `print_to_stdstream` is called.

Reply:
good catch. Updated